README.md

```diff
@@ -119,3 +119,21 @@ adaptor.get_by_id(document_id="id:YOUR_NAMESPACE:YOUR_SCHEMA_NAME::SOME_DOCUMENT
 ```
 
 All of the above search functionality assumes that a valid set of vespa credentials is available in `~/.vespa`, or in a directory supplied to the `VespaSearchAdapter` constructor directly. See [the docs](docs/vespa-auth.md) for more information on how vespa expects credentials.
+
+# Test setup
+Some tests rely on a local running instance of vespa.
+
+This requires the [vespa cli](https://docs.vespa.ai/en/vespa-cli.html) to be installed.
+
+Setup can then be run with:
+
+```
+make vespa_setup
+make test
```

Comment:
this broke on my machine as i hadn't installed the `vespa` extra - is it worth adding the install command that sits in CI somewhere into this README (maybe here)?

this one:
```
poetry install --all-extras --with dev
```

Reply:
Good shout